# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/cairo/cairo/src/cairo-win32.h
# Commit: 3cdfb39376b7
# Full Hash: 3cdfb39376b736597fe658a322389117686b7e82
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2024-05-07 09:13:07
# Regressor Bug: 1892913
# File Overlap Count: 3
# Description:
#   Bug 1892913 - patch 1 - Update cairo source to 1.18.0 r=gfx-reviewers,lsalzman
#   
#   Wholesale replacement of files in gfx/cairo/cairo with contents of the 1.18.0 release tarball,
#   omitting subdirectories that are not part of the core library build.
#   
# ==============================================================================

diff -r 9daa351f875b -r 3cdfb39376b7 gfx/cairo/cairo/src/cairo-win32.h
--- a/gfx/cairo/cairo/src/cairo-win32.h	Mon May 06 19:22:35 2024 +0000
+++ b/gfx/cairo/cairo/src/cairo-win32.h	Mon May 06 19:25:12 2024 +0000
@@ -69,15 +69,9 @@
 cairo_public HDC
 cairo_win32_surface_get_dc (cairo_surface_t *surface);
 
-cairo_public HDC
-cairo_win32_get_dc_with_clip (cairo_t *cr);
-
 cairo_public cairo_surface_t *
 cairo_win32_surface_get_image (cairo_surface_t *surface);
 
-cairo_public cairo_status_t
-cairo_win32_surface_get_size (const cairo_surface_t *surface, int *width, int *height);
-
 #if CAIRO_HAS_WIN32_FONT
 
 /*
@@ -111,33 +105,8 @@
 cairo_win32_scaled_font_get_device_to_logical (cairo_scaled_font_t *scaled_font,
 					       cairo_matrix_t *device_to_logical);
 
-cairo_public BYTE
-cairo_win32_get_system_text_quality (void);
-
 #endif /* CAIRO_HAS_WIN32_FONT */
 
-#if CAIRO_HAS_DWRITE_FONT
-
-/*
- * Win32 DirectWrite font support
- */
-cairo_public cairo_font_face_t *
-cairo_dwrite_font_face_create_for_dwrite_fontface (void *dwrite_font, void *dwrite_font_face);
-
-void
-cairo_dwrite_scaled_font_set_force_GDI_classic (cairo_scaled_font_t *dwrite_scaled_font, cairo_bool_t allowed);
-
-cairo_bool_t
-cairo_dwrite_scaled_font_get_force_GDI_classic (cairo_scaled_font_t *dwrite_scaled_font);
-
-void
-cairo_dwrite_set_cleartype_params (FLOAT gamma, FLOAT contrast, FLOAT level, int geometry, int mode);
-
-int
-cairo_dwrite_get_cleartype_rendering_mode ();
-
-#endif /* CAIRO_HAS_DWRITE_FONT */
-
 CAIRO_END_DECLS
 
 #else  /* CAIRO_HAS_WIN32_SURFACE */